lib/matplotlib/tests/test_dates.py

```diff
@@ -128,6 +128,23 @@ def test_RRuleLocator():
     fig.autofmt_xdate()
 
 
+def test_RRuleLocator_dayrange():
+    ret = 0
+
+    try:
```

Comment:
You do not need the `try...except` here.  If it raises, then the test will fail (but the exception will be caught and the rest of the test suite will run).

Just a comment about how this is just testing 'does not raise' is enough.